test/contrib/s3_test.py

```diff
@@ -314,6 +314,15 @@ def test_get(self):
         self.assertEquals(content, self.tempFileContents.decode("utf-8"))
         tmp_file.close()
 
+    def test_get_as_bytes(self):
+        create_bucket()
+        s3_client = S3Client(AWS_ACCESS_KEY, AWS_SECRET_KEY)
+        s3_client.put(self.tempFilePath, 's3://mybucket/putMe')
+
+        contents = s3_client.get_as_bytes('s3://mybucket/putMe')
+
+        self.assertEquals(contents, self.tempFileContents)
+
```

Comment:
Could you also add 2 other get_as_string tests? Maybe one for utf-8 and one for another encoding?

Reply:
Done